components/ImpactCard.jsx

```diff
@@ -5,22 +5,28 @@ import { media } from "../utils";
 /*
     Example:
     - Thumbnail: image
-    - impactNum: some number
+    - isImageTop: will put the right order of image and text as specified
+    - impactNum: some number or title of the entire card
     - Description: text (cool statistic caption)
+    - textColour: color of the title and description in card
+    - textSize: size of text at the very top (title)
     - imageHeight and imageWidth are dimensions of thumbnail
 */
 export const ImpactCard = ({
   thumbnail,
+  isImageTop,
```

Comment:
you SHOULD put a default prop here because you've already used this before on another page. you will break if it isn't specified on the home page. ```isImageTop = true```

Reply:
you fixed it in the later file but in case you did not code the other section, you would be breaking the other components because everyone else assumes true. the same is said for the fontSize

Reply:
so writing a value here gives it a default value that will be overwritten if a value is specified?